extensions/wikia/WikiaMobile/WikiaMobileCategoryService.class.php

```diff
@@ -80,6 +80,7 @@ public function alphabeticalList() {
 			$title = $categoryPage->getTitle();
 			$category = Category::newFromTitle( $title );
 			$collections = $this->model->getCollection( $category, $format );
+			ksort( $collections['items'] );
```

Comment:
Is `ksort` - i18n-capable? Do we want the list to be sorted according to proper alphabet? @Warkot